lib/icinga/pluginutility.cpp

```diff
@@ -217,3 +236,32 @@ String PluginUtility::FormatPerfdata(const Array::Ptr& perfdata)
 
 	return result.str();
 }
+
+Value PluginUtility::LookupPlugin(const String& executable)
```

Comment:
Can this method ever actually return anything other than a `String`? Its return type should probably be `String` instead.